scipy/stats/_continuous_distns.py

```diff
@@ -3247,6 +3247,142 @@ def _ppf(self, q, k, s):
 mielke = mielke_gen(a=0.0, name='mielke')
 
 
+class kappa4_gen(rv_continuous):
+    """Kappa 4 parameter distribution
```

Comment:
Nitpick: full stop at the end of line (otherwise, the doc builder gets confused IIRC)